glusterd2/device/deviceres.go

```diff
@@ -0,0 +1,21 @@
+package device
+
+import "github.com/pborman/uuid"
+
+const (
+	// DeviceEnabled represents enabled
+	DeviceEnabled = "Enabled"
+
+	// DeviceFrozen represents frozen
+	DeviceFrozen = "Frozen"
+
+	// DeviceEvacuated represents evacuated
+	DeviceEvacuated = "Evacuated"
+)
+
+// Info is the added device info
+type Info struct {
+	PeerID uuid.UUID `json:"peerid"`
```

Comment:
Name the json string as `peer-id` to be consistent with existing naming convention